Source/Cmlx/mlx-conditional/accelerate-softmax.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright © 2024 Apple Inc.
+
+// Note: this stubs out accelerate/softmax on x86 (e.g. via Release builds)
+
+#if defined(__aarch64__)
+
+#include "../mlx/mlx/backend/accelerate/softmax.cpp"
+
+#else
+
+#include "mlx/primitives.h"
+
+namespace mlx::core {
+    void Softmax::eval_cpu(const std::vector<array>& inputs, array& out) {
+        throw std::invalid_argument(
+            "Softmax is not available on x86 cpu");
+    }
```

Comment:
Why not use `Softmax::eval` here and support it on x86?

Reply:
Sure, that is a good idea (though unwise to actually call :-) )